src/app/(profile)/my-profile/(components)/CreateProfileForm/CreateProfileForm.tsx

```diff
@@ -125,7 +125,9 @@ const CreateProfileForm = () => {
 
         uploadedFileUrl && (await updateUserAvatar(uploadedFileUrl))
 
-        await createProfile(payload)
+        const createdProfile = await createProfile(payload)
+        createdProfile &&
+          updateSession({ ...session.user, name: payload.fullName })
```

Comment:
**issue:** We are missing `profileId` here, and user will have profileId as null even if he has profile created

Reply:
done in https://github.com/nerdbord/good-dev-hunting-app/pull/263/commits/3f28a890f5f76461c3c8546f281a796b84b202ec